CHANGELOG.md

```diff
@@ -42,6 +42,14 @@ for Rust libraries in [RFC #1105](https://github.com/rust-lang/rfcs/blob/master/
   have been. All types in Diesel are now correctly only considered
   non-aggregate if their parts are.
 
+* Diesel CLI will now look for `diesel.toml` to determine the project root
+  before looking for `Cargo.toml`.
+
+* Any relative paths in `diesel.toml` will now be treated as relative to the
+  project root (the directory containing either `diesel.toml` or `Cargo.toml`).
```

Comment:
I think that this clarification makes it a little more confusing. It might be more effective to state in the previous paragraph what the project root is, then just reuse the term, especially since there can be multiple `Cargo.toml` files (which this PR is trying to fix anyways).